source/_patterns/01-molecules/tag-list/tag-list~article-type.yaml

```diff
@@ -0,0 +1,17 @@
+list:
+  articleType: true
```

Comment:
This needs some other name, as 'article type' has meaning different to this (which is a list of details/metadata about the item, which _includes_ the type). See also https://github.com/libero/pattern-library/pull/57#discussion_r258521878.

Reply:
As discussed with @thewilkybarkid we have decided for the moment to go for itemDetails instead of articleType as it's closer to what is needed.